menyer/kackerlackan/index.html

```diff
@@ -0,0 +1,100 @@
+<!DOCTYPE html>
+<html lang="sv">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Kackerlackan</title>
+    <link rel="stylesheet" href="stylesheet.css">
+</head>
+
+<body>
+    <header id="Header">
```

Comment:
If one only is using one `<header>` then `id="Header"`  is redundant. Consider removing `id="Header"` and just select on `header`.